orttraining/orttraining/python/training/ortmodule/torch_cpp_extensions/_aten_op_factory.py

```diff
@@ -0,0 +1,124 @@
+# -------------------------------------------------------------------------
```

Comment:
This file is not a PyTorch CPP extension, as described here https://pytorch.org/docs/stable/cpp_extension.html

I would prefer moving this file outside torch_cpp_extensions as it can interfere with onnxruntime packaging

maybe `orttraining/orttraining/python/training/ortmodule/_aten_op_symbolics_factory.py`?